polaris/benchmark/_base.py

```diff
@@ -418,70 +434,15 @@ def evaluate(
         # Instead of having the user pass the ground truth, we extract it from the benchmark spec ourselves.
         # This simplifies the API, but also was added to make accidental access to the test set targets less likely.
         # See also the `hide_targets` parameter in the `Subset` class.
-        test = self.get_train_test_split()[1]
-
-        if not isinstance(test, dict):
-            test = {"test": test}
-
-        y_true = {}
```

Comment:
All of this logic beginning here has been moved to `evaluate_benchmark` in `polaris/evaluate/utils.py`. The functionality is unchanged, with some minor refactoring to account for the possibility that `y_prob` may be `None`.